data-esnext.js

```diff
@@ -778,6 +778,7 @@ exports.tests = [
     babel: true,
     jsx: true,
     chrome58: 'flagged',
+    typescript: typescript.corejs,
```

Comment:
@ljharb TS doesn't require `core.js` for that
https://www.typescriptlang.org/play/index.html#src=let%20%7B%20x%2C%20y%2C%20...z%20%7D%20%3D%20%7B%20x%3A%201%2C%20y%3A%202%2C%20a%3A%203%2C%20b%3A%204%20%7D%3B%0Alet%20n%20%3D%20%7B%20x%2C%20y%2C%20...z%20%7D%3B